EIPS/eip-5453.md

```diff
@@ -14,7 +14,7 @@ requires: 165, 712, 1271, 5750
 ## Abstract
 
 This EIP establish a general protocol for permitting approving function calls in the same transaction rely on [EIP-5750](./eip-5750.md).
-Unlike a few prior art ([EIP-2612](./eip-2612.md) for [EIP-20](./eip-20.md), [EIP-4494](./eip-4494.md) for [EIP-721](./eip-721.md) that
+Unlike a few prior art ([EIP-2612](./eip-2612.md) for [EIP-20](./eip-20.md), `EIP-4494` for [EIP-721](./eip-721.md) that
```

Comment:
Art? Why have you unlinked EIP-4494?

Reply:
Because EIP-4944 was still a DRAFT. the linter complains if we attempt to move EIP-5453 to REVIEW until 4944 is moved to REVIEW

Reply:
Ah. @SamWilsn is it okay to override the bot here?

Also, I still don't get what you mean by art.

Reply:
> @SamWilsn is it okay to override the bot here?

I don't believe so. Just don't mention EIP-4944, or move it to the correct stage. If it's important enough to mention, it's important enough to make sure it doesn't change out from under you.

> Also, I still don't get what you mean by art.

"Prior Art" is a legal term meaning related work that came before.